recipes/librmm/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "librmm" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: eddd5b35b016d665eb4e8ea2dd31497d913b3e1eb831124e4cb27cba747267a0
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux)]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.14.0
+    - make
+  host:
+    - spdlog =1.7.0
```

Comment:
Should this be a `run` requirement as well?

Reply:
I believe we're using it as a header-only library, but @kkraus14 can confirm.

Reply:
Yes it should be: https://github.com/rapidsai/rmm/blob/branch-0.20/conda/recipes/librmm/meta.yaml#L38

Reply:
Thanks for the clarification. So it should be `run` only. Included a code suggestion below...

Reply:
Thanks, fixed in https://github.com/conda-forge/staged-recipes/pull/14749/commits/ae4496c604ee1642817aebf1b1d47ed2f9bfa4bb